[Doc] Create a contact section and cleanup the introduction (#6613)
PyCQA/pylint: doc/contact/contact.rst

Bug reports, feedback
---------------------
.. _bug reports, feedback:

You think you have found a bug in Pylint? Well, this may be the case

> since Pylint and python are under heavy development!

Reviewer: ```suggestion
since Pylint and Python are under heavy development!
```

Please take the time to check if it is already in the issue tracker at
https://github.com/PyCQA/pylint

> Note that if you don't find something you have expected in Pylint's issue
> tracker, it may be because it is an issue with one of its dependencies, namely
> astroid:

Reviewer: ```suggestion
Note that the issue might also be reported in one of Pylint's major dependencies,
astroid:
```
